tools/gcp-ips/main.go

```diff
@@ -30,17 +30,19 @@ import (
 	"path/filepath"
 	"sort"
 	"strings"
-	"sync"
 
 	"github.com/olekukonko/tablewriter"
 	"golang.org/x/net/context"
 	"golang.org/x/oauth2/google"
+	"golang.org/x/sync/semaphore"
 	"google.golang.org/api/compute/v1"
 )
 
 const (
 	// outputDir is the name of the directory where output files will be written.
 	outputDir = "output"
+	// maxWorkers is the max number of goroutines allowed to run in parallel
+	maxWorkers = int64(32)
```

Comment:
A `const` in Go doesn't have a type until it's used in place, so you shouldn't specify the type here. It should just be `maxWorkers = 32`. It can then be later compared against an int64, int, int32, uint32, etc without having to cast it.